src/auto-trace.helper.spec.js

```diff
@@ -78,122 +107,4 @@ describe('auto-trace.js', () => {
 			expect(appendExtraContext(err, extraContext)).toEqual(Error(`Something went wrong! Extra Context: {"userid":23,"moreInfo":"junk"}`));
 		});
 	});
-
-	describe('removeAutoTraceFromErrorStack', () => {
```

Comment:
We no longer regex items out of the stack, instead we use `createError()` when new errors are created